net-irc/quassel/files/quassel-0.14.0-fix-desktop-application-name.patch

```diff
@@ -0,0 +1,54 @@
+From 020c163421691fa37330826df92ac0a248721290 Mon Sep 17 00:00:00 2001
```

Comment:
Please add a link to the PR at the top of https://github.com/quassel/quassel/pull/615